src/dex-ui/pages/dao/ProposalDetailsPage/GovernanceProposalConfirmationDetails.tsx

```diff
@@ -61,11 +64,13 @@ export function GovernanceProposalConfirmationDetails(props: GovernanceProposalC
     executeProposal,
     cancelProposal,
     status,
+    state,
     hasConnectedWalletVoted,
     isAuthor,
   } = props;
 
   const contractId = proposal?.contractId ?? "";
+  const isVotingDisabled = !proposal || Number(votingPower) <= 0;
```

Comment:
`Number(votingPower) ` could resolve to `NaN`. We need a guard to cover for that case.

Reply:
Added the check to check for NaN type too.